src/status_im/common/contact_list/view.cljs

```diff
@@ -1,10 +1,16 @@
 (ns status-im.common.contact-list.view
   (:require
     [quo.core :as quo]
+    [quo.theme]
     [react-native.core :as rn]
     [status-im.common.contact-list.style :as style]))
 
+(defn contacts-section-footer
+  []
+  [rn/view style/contacts-section-footer])
+
 (defn contacts-section-header
-  [{:keys [title index]}]
-  [rn/view (style/contacts-section-header (= index 0))
-   [quo/divider-label title]])
+  [{:keys [title]}]
+  (let [theme (quo.theme/use-theme-value)]
```

Comment:
im not sure this is efficient, could you elaborate on why the header has a different color?

Reply:
The header has to have a background colour set when we want to use it as a sticky header.

If we do not set a background colour, we get this overlap of the header and the items when the header sticks to the top

![Simulator Screen Recording - iPhone 13 - 2024-03-01 at 06 15 17](https://github.com/status-im/status-mobile/assets/45393944/b772bc90-2f65-4168-a0a8-7ab47924f649)

Reply:
as we have our own implementation of section-list do you think if would be possible to set the theme as a prop to section-list and pass it to the header?